engine/src/flutter/lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -126,6 +126,7 @@ extension type CanvasKit(JSObject _) implements JSObject {
   external SkTypefaceFactory get Typeface;
 
   external double GetWebGLContext(DomHTMLCanvasElement canvas, SkWebGLContextOptions options);
+  @JS('GetWebGLContext')
```

Comment:
shouldn't this move UP ☝️

Reply:
No, the method on line 128 already has the right name. This is to rename the method on line 130 so we use the same JS method but with different arg types.

Reply:
Oh! Okay...